fabric-api-bom/build.gradle

```diff
@@ -0,0 +1,31 @@
+plugins {
+	id 'java-platform'
+}
+
+version = rootProject.version
+
+publishing.publications {
+	register('mavenJava', MavenPublication) {
+		from(components['javaPlatform'])
+	}
+}
+
+tasks.withType(GenerateModuleMetadata) {
+	// todo: RemoteSignJar fails when this is false (as set by parent build script)
```

Comment:
Is this something you want me to look into? I forget the reason why I disabled it on the parent projects as well.

Reply:
I think it's probably fine, I didn't notice any issues resulting from the metadata being present. Just left the notes as it does differ from the configuration done to the other subprojects.